protos/flyteidl/plugins/ray.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
+
+package flyteidl.plugins;
+
+option go_package = "github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/plugins";
+
+import "google/protobuf/timestamp.proto";
+
+// RayJobSpec defines the desired state of RayJob
+message RayJob {
+  // RayClusterSpec is the cluster template to run the job
+  RayCluster ray_cluster = 1;
+  // runtime_env is base64 encoded.
+  // Ray runtime environments: https://docs.ray.io/en/latest/ray-core/handling-dependencies.html#runtime-environments
+  string runtime_env = 2;
+  // shutdown_after_job_finishes will determine whether to delete the ray cluster once rayJob succeed or failed.
+  bool shutdown_after_job_finishes = 3;
```

Comment:
doesnt this seem odd to be encoded into the request? Should this be implicit?

Reply:
removed it, I think we can move it to propeller's config